Sources/KlaviyoSwift/KlaviyoAPI.swift

```diff
@@ -103,11 +103,10 @@ extension KlaviyoAPI.KlaviyoRequest {
     var url: URL? {
         switch endpoint {
         case .createProfile, .createEvent, .registerPushToken, .unregisterPushToken:
-            if #available(iOS 17.0, *) {
-                return URL(string: "\(environment.analytics.apiURL)/\(path)/?company_id=\(apiKey)", encodingInvalidCharacters: false)
-            } else {
+            if !environment.analytics.apiURL.isEmpty {
```

Comment:
Not exactly the same thing as before but it'll work! The compile time check didn't work out huh?

Reply:
Yeah, I added why I went this way in the description.